makefile

```diff
@@ -12,13 +12,20 @@ build:
 	swift build -c release
 
 build-cli:
-	swift build --product apollo-ios-cli -c release
+	swift build --product apollo-ios-cli -c release; \
+	cp -f .build/release/apollo-ios-cli apollo-ios-cli
 
 build-cli-universal:
-	swift build --product apollo-ios-cli -c release --arch arm64 --arch x86_64
+	swift build --product apollo-ios-cli -c release --arch arm64 --arch x86_64; \
+	cp -f .build/apple/Products/Release/apollo-ios-cli apollo-ios-cli
 
 build-cli-for-cocoapods:
 	swift build --product apollo-ios-cli -c release -Xswiftc -DCOCOAPODS
```

Comment:
We don't copy the built binary for CocoaPods workflows because that happens in the podspec `prepare_command` command.